.gitignore

```diff
@@ -7,3 +7,18 @@ __pycache__
 *.egg*
 
 .DS_Store
+
+wandb
+
+
+legacy/test_automl/data
+legacy/test_automl/test.py
+*.pkl
+*.pt
+openfe-singlecell
+examples/tuning/cta_svm/test
+examples/tuning/cta_svm/train
+examples/tuning/cta_svm/map
+test
+train
+map
```

Comment:
Please do not ignore these (and the legacy stuff above). Mostly because its unmaintainable. **YOU** should be the one keeping track of them and decide whether you want to ignore them, as these are you local dev stuff.